tests/unit/api/main.py

```diff
@@ -1185,29 +1188,42 @@ class Test_switch_register:
     def test_basic(self):
         """Calling switch_register should create an object in the db."""
         api.switch_register('sw0', type=MOCK_SWITCH_TYPE,
-                username="switch_user", password="switch_pass", hostname="switchname")
+                            username="switch_user",
+                            password="switch_pass",
+                            hostname="switchname")
         assert model.Switch.query.one().label == 'sw0'
 
     def test_duplicate(self):
-        """switch_register should complain if asked to make a duplicate switch."""
+        """
+        switch_register should complain if asked to make a duplicate switch.
+        """
```

Comment:
Also, I believe we actually have a good number of pep257 violations throughout the source, mostly violating the "fits on one line" requirement for the summary. I don't think we need to fix those in this pr though, esp. since we don't have a linter for it, and in a lot of cases it's going to require rewording things, which requires a bit of though on a case by case basis.